CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [14.1.2] - 2023-05-24
+
+### Changes
+
+-   Email templates updated to support superhuman client.
```

Comment:
"Update email templates to fix an issue with styling on some email clients"

1. We arent sure if the issue is exclusive to SuperHuman
2. Be mindful of how you write CHANGELOG copies, this is public